setup.cfg

```diff
@@ -11,7 +11,8 @@ minversion = 1.8
 envlist =
     docs,
     flake8,
-    py{36,37,38,39,310}-django{22,30,31,32}
+    py{36,37,38,39,310}-django{22,30,31,32},
+    py{38,39,310}-django{40}
```

Comment:
@ethanfangwsp This should be one line, add the `40` to the django matrix

Reply:
This is because django 4.0 only supports 3.8-3.10